ext/-test-/arith_seq/beg_len_step/beg_len_step.c

```diff
@@ -0,0 +1,19 @@
+#include "ruby/ruby.h"
+
+static VALUE
+arith_seq_s_beg_len_step(VALUE mod, VALUE obj, VALUE len, VALUE err)
+{
+  VALUE r;
+  long begp, lenp, stepp;
+
+  r = rb_arithmetic_sequence_beg_len_step(obj, &begp, &lenp, &stepp, NUM2LONG(len), NUM2INT(err));
+
+  return rb_ary_new_from_args(4, r, LONG2NUM(begp), LONG2NUM(lenp), LONG2NUM(stepp));
```

Comment:
I want to remove the suffix `p` from `begp`, `lenp`, and `stepp` because they are not pointers.

```suggestion
  long beg, len, step;

  r = rb_arithmetic_sequence_beg_len_step(obj, &beg, &len, &step, NUM2LONG(len), NUM2INT(err));

  return rb_ary_new_from_args(4, r, LONG2NUM(beg), LONG2NUM(len), LONG2NUM(step));
```